fix(rdb): pub/sub MTA-3599 (#2141)
scaleway/docs-content: managed-databases/redis/api-cli/using-pub-sub-feature.mdx

---
meta:
  title: Using Publish/Subscribe messaging with Redis™
  description: Learn more about the the Publish/Subscribe feature with Redis™
content:
  h1: Using Publish/Subscribe messaging with Redis™
  paragraph: Learn more about the the Publish/Subscribe feature with Redis™

> tags: databases redis pub/sub messaging broker python

Reviewer: python?

categories:
  - managed-databases
  - redis
---

> The Pub/Sub feature is available with Redis™ 6 and 7.

Reviewer 2: est-ce qu'on expliquerait pas que Publish/Subscribe c'est aka "Pub/Sub" au lieu de commencer directement avec ca

Author: ```suggestion
The Publish/Subscribe feature is available with Redis™ 6 and 7.
```

Pub/Sub is a highly scalable messaging model fit to handle large amounts of data and that enables asynchronous communication on low bandwidth.

The model decouples **the publishers**, the clients sending a message, from **the subscribers**, the one or several clients receiving the message. Messages are published to a third component, the message **broker**, whose job is to forward messages from publishers to subscribers under **topics**.

> Topics describe the data contained within messages, allowing subscribers to choose which messages to receive based on the subject of the topic. For example, a under a topic named `/weather-paris` one could expect to find data about the weather in Paris.

Reviewer 3: ```suggestion
Topics describe the data contained within messages, allowing subscribers to choose which messages to receive based on the subject of the topic. For example, under a topic named `/weather-paris` one could expect to find data about the weather in Paris.
```

> To implement Pub/Sub system using Redis, you can run commands such as `PUBLISH` and `SUBSCRIBE`.

Author: ```suggestion
To implement Pub/Sub using Redis, you can run commands such as `PUBLISH` and `SUBSCRIBE`.
```

> To implement Pub/Sub system using Redis, you can run Python commands such as `PUBLISH` and `SUBSCRIBE`.

Reviewer: This is not python commands. Those are built-in Redis commands
